RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -1408,12 +1407,16 @@ initializeProtoCands(std::list<PFEGammaAlgo::ProtoEGObject>& egobjs) {
 	     }
 	   }// loop over tracks in primary vertex
 	    // if associated to good non-GSF matched track remove this cluster
-	   if( Algo < reco::TrackBase::pixelLessStep && nexhits == 0 && fromprimaryvertex ) {
+	   if( PFTrackAlgoTools::isNotPixelLessStep(trackref->algo()) && nexhits == 0 && fromprimaryvertex ) {
```

Comment:
the name is confusing
tobtec etc are "not isNotPixelLessStep" in this case.
It's better to call it by purpose rather than by the actual implementation:
isGoodForEGMPrimary or something like that